src/WebPages/WishListPage/Wishlist.css

```diff
@@ -0,0 +1,56 @@
+@import url(https://wanderer-store.netlify.app/main.css);
+@import url(https://wanderer-store.netlify.app//index.css);
+/* @import url(https:wanderer-store.netlify.app/WebPages/productListing/productlist.css); */
+
+body {
+  font-family: roboto;
+  color: var(--text-color);
+}
+
+.wishlist-product-container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 2rem;
+  padding: 2rem 0 0 4rem;
+  margin-bottom: 2rem;
+}
+
+.vertical-card-component {
+  max-width: 18rem;
+}
+
+.vertical-card-component:hover {
+  box-shadow: var(--shadow-box);
+}
+
+.heart-icon {
+  /* color: red; */
+  border-radius: 50%;
+}
+
+.wishlist-card {
+  max-width: 18rem;
+}
+
+.icons {
+  padding: 0 1rem;
+}
+
+@media screen and (max-width: 1200px) {
+  .wishlist-product-container {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+@media screen and (max-width: 800px) {
+  .wishlist-product-container {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .wishlist-product-container {
+    grid-template-columns: 1fr;
+  }
+}
```

Comment:
I would suggest using mobile first approach if you have to use media queries multiple times. 
UI can be easily responsive, with mobile first approach as it reduces dependency on media queries.
This also removes repetition from code.

Reply:
Thanks shrey for the comment ! I am planning to work on the responsiveness after completing the logic part . I hope that's fine